### DISCUSSION TITLE
Would there be interest in having a community repository for rule definitions of popular systems?

### DISCUSSION THREAD

**Discussion by mainrs** [hlorenzi/customasm#155] [Ideas]
I've seen a lot of z80 questions. I am sure most of them just try to use the tool and get familiar for it. But having something to compare it to would be a cool idea!

If one starts a repository, would you be willing to maybe link it in the readme?

**Comment by hlorenzi** [comment]
Yeah, I'd love to link to other people's implementations! But maybe we could make a community page on the wiki, though? We can have a list of repos for each target platform. I think the wiki should already be set to be editable by anyone, in fact.
